app/src/main/java/com/github/se/eventradar/ui/chat/Chat.kt

```diff
@@ -264,7 +250,6 @@ fun ChatScreenPreview() {
   val sampleUiState =
```

Comment:
Suggestion: The ChatScreenPreview() can be updated.
View this example from `Home.kt`:
```
@Preview(showBackground = true, showSystemUi = true)
@ExcludeFromJacocoGeneratedReport
@Composable
fun HomeScreenPreview() {
  val mockEventRepo = MockEventRepository()
  val mockUserRepo = MockUserRepository()
  HomeScreen(
      EventsOverviewViewModel(mockEventRepo, mockUserRepo),
      NavigationActions(rememberNavController()))
}
```